.ci/jenkins/Jenkinsfile.release

```diff
@@ -377,20 +373,21 @@ pipeline {
         }
         stage ('Send email to BSIG') {
             steps {
-                emailext body: 'The community ${kieVersion} was released. \n' +
-                ' \n' +
-                'The tags are pushed and the binaries for the webs will be uploaded soon to filemgmt.jboss.org. \n' +
-                ' \n' +
-                ' \n' +
-                'You can download the artifacts..: \n' +
-                ' \n' +
-                'business-central artifacts: https://repository.jboss.org/nexus/content/groups/public-jboss/org/kie/business-central/$kieVersion/ \n' +
-                'business-central-webapp: https://repository.jboss.org/nexus/content/groups/public-jboss/org/kie/business-central-webapp/$kieVersion/ \n' +
-                'business-monitoring-webapp: https://repository.jboss.org/nexus/content/groups/public-jboss/org/kie/business-monitoring-webapp/$kieVersion/ \n' +
-                'jbpm-server-distribution (single zip): https://repository.jboss.org/nexus/content/groups/public-jboss/org/kie/jbpm-server-distribution/$kieVersion/ \n' +
-                '\n' +
-                'Component version:\n' +
-                'kie = $kieVersion', subject: 'community-release-$baseBranch $kieVersion was released', to: 'bsig@redhat.com'
+                emailext body: "\n" +
```

Comment:
The same here
```
                emailext body: """
The community ${kieVersion} was released. 

The tags are pushed and the binaries for the webs will be uploaded soon to filemgmt.jboss.org.


You can download the artifacts..: 

business-central artifacts: https://repository.jboss.org/nexus/content/groups/public-jboss/org/kie/business-central/$kieVersion/ 
business-central-webapp: https://repository.jboss.org/nexus/content/groups/public-jboss/org/kie/business-central-webapp/$kieVersion/
business-monitoring-webapp: https://repository.jboss.org/nexus/content/groups/public-jboss/org/kie/business-monitoring-webapp/$kieVersion/
jbpm-server-distribution (single zip): https://repository.jboss.org/nexus/content/groups/public-jboss/org/kie/jbpm-server-distribution/$kieVersion/

Component version:
kie = $kieVersion""", subject: "community-release-$baseBranch $kieVersion was released", to: ',bsig@redhat.com'
```

Reply:
is `to: ',bsig@redhat.com'` ok? shouldn't it be `to: 'bsig@redhat.com'`?

Reply:
fixed